config.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+/* eslint no-process-env: 0 */
+
+const env = process.env.NODE_ENV || 'production';
+
+module.exports = {
+  PRETTY_DATE_FORMAT: 'DD MMMM YYYY',
+  dateTimeFormat: 'DD MMM YYYY HH:mm:ss',
+  env: env,
+  govukNotify: {
+    notifyApiKey: process.env.NOTIFY_KEY,
+    caseworkerEmail: process.env.FEEDBACK_INBOX,
+    userConfirmationTemplateId: process.env.FEEDBACK_TEMPLATE_ID
```

Comment:
I would suggest to rename `userConfirmationTemplateId` to avoid confusion,
maybe something like `feedbackSubmissionTemplateId`

Reply:
missed that, my bad